__tests__/app.test.js

```diff
@@ -50,3 +50,35 @@ describe("GET categories", () => {
       });
   });
 });
+
+describe("GET review by id", () => {
+  test("200: should respond with an object that contains the relavant review properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body).toBe("object");
+        expect(res.body.reviewToSend).toEqual(
+          expect.objectContaining({
+            review_id: expect.any(Number),
```

Comment:
I would recommend that you check this object is coming back exactly as review number 1, rather than expect.any() statements, so for example check that review_id: 1, title is the actual title as a string, review_body is the actual body and so on. This will give you much better confidence that the test worked and returned the exact review number 1 you were expecting to see. Please amend this test.